MainWindow.h

```diff
@@ -99,7 +100,36 @@ class MainWindow : public QMainWindow {
  private:
   static MainWindow *m_instance;
 
+    // Coffee : set nodeResource as pointer. Clear when opening new file!
   QHash<buffers::TreeNode *, QMdiSubWindow *> subWindows;
+  const QList<QVariant> defaultGroupNames = {
+    tr("Sprites"),
+    tr("Sounds"),
+    tr("Backgrounds"),
+    tr("Paths"),
+    tr("Scripts"),
+    tr("Shaders"),
+    tr("Fonts"),
+    tr("Time Lines"),
+    tr("Objects"),
+    tr("Rooms"),
+    tr("Includes"),
+    tr("Configs")
+  };
+  const QList<TypeCase> defaultGroupTypes = {
+    TypeCase::kSprite,
+    TypeCase::kSound,
+    TypeCase::kBackground,
+    TypeCase::kPath,
+    TypeCase::kScript,
+    TypeCase::kShader,
+    TypeCase::kFont,
+    TypeCase::kTimeline,
+    TypeCase::kObject,
+    TypeCase::kRoom,
+    TypeCase::kInclude,
+    TypeCase::kSettings
+  };

```

Comment:
This is redundant and already exists. Also it's noisy, so condense it.
https://github.com/enigma-dev/RadialGM/blob/fa091deac99ceae4fb12dcd74f193cdeb31dcb62/MainWindow.cpp#L300